UI/Tests/Test.cs

```diff
@@ -73,5 +73,19 @@ public void CheckNotLoggedWarningOnEditPage()
             CustomLogger.LogInfo(LogLevel.Info, "Verify warning for not logged user displayed");
             Assert.That(IsNotLoggedWarningDisplayed, Is.True);
         }
+
+        [Test]
+        [Category("Display")]
+        public void DisplayAboutPageTest_Success()
+        {
+            var aboutPage = MainPage
+                .ToLoginPage()
```

Comment:
Place some logger

Reply:
Is it Ok if I use the same approach like Yevgeniy in actual testcases instead of method chaining approach?